### ISSUE TITLE
Generated ID is different when using Add-Mytask, and then Get-MyTask..

### ISSUE THREAD

**Issue by KurtDeGreeff** [jdhitsolutions/MyTasks#1]
When I create a new task it looks ID 0 is always used by Add-MyTask
Only after using Get-MyTask you see 'correct' ID.
Can Add-MyTask by default not better show the correct ID..?

**Comment by jdhitsolutions** [comment]
I understand what you're saying. I think I have an idea that might solve this. The ID number is generated at runtime. Much like PowerShell jobs, there's no guarantee you will have the same ID number.

**Comment by jdhitsolutions** [comment]
I pushed a new version so that when you use -Passthru you get the correct task object

**Comment by KurtDeGreeff** [comment]
Okay, many thanks!
I was wondering..besides using Write-Host, is it true to say that everything I see in Powershell output screen comes from the pipeline? In other words can I send output to the screen without the Powershell pipeline..? Thx

**Comment by jdhitsolutions** [comment]
Write-Host does not send anything to the pipeline.  But you can colorize the output.  Otherwise, yes, everything else goes through the pipeline.

**Comment by jdhitsolutions** [comment]
I'm considering a way to still write to the pipeline but with colorized output.
